output/_note.md

```diff
@@ -0,0 +1 @@
+Images will be output here.
```

Comment:
Keeps the output folder in the product, so that people aren't wondering why they aren't seeing the downloads.